python-setup/python-setup-suggestions.qmd

```diff
@@ -0,0 +1,154 @@
+
+Whenever you _re_open a folder in Positron, Positron will activate the environment you used the last time you worked in the folder. If this is your first time opening the folder, Positron will make an educated guess about which environment to use or ask you to select an environment. In that case, the button will display Start Interpreter.
+
+![Start interpreter](images/start-interpreter.png)
+
+## How to change your environment
+
+To change the environment, open the drop down menu and click the three dots next to the current environment's name. Positron will list the environments it detects on your machine. Select the one you want to use.
+
+![Enviornments list](images/list.png){width=300}
+
+But what if you want to create a new environment specific to your project? 
+
+## Create a new environment
+
+A good best practice is to create a new environment for each new project. This forces you to be intentional about your environment, but it also protects your other projects. If you run each project in the same global environment, then any changes you make with one project will affect other projects. Not so if each project uses its own environment.
+
+So how do you do it? To create a new environment:
+
+1. Open the Command Palette with **Command + Shift + P** (Mac) or **Control + Shift + P** (Windows and Linux)
+
+![Command Palette](images/palette.png)
+
+2. Select the **Python: Create Environment** command.
+
+3. Then select the type of environment you would like to create: Venv or Conda.
+
+![Select type](images/type.png)
+
+4. Next select the Python interpreter (venv) or Python version (conda) that you would like to use for the environment.
+
+![Select interpreter](images/interpreter.png)
+
+Positron will create the environment in you current folder and then activate it for use. Along the way, it will do several things:
+
+- It will install any package dependencies listed in a `requirements.txt`, `pyproject.toml`, or `environment.yml` file that appears in your folder. This is helpful if you've cloned your project from a git collaborator and wish to quickly recreate their environment.
+- It will add your environment directory to your folder's `.gitignore` file, so you will be less likely to commit your environment to github. More on that below.
+
+Now that you have an environment, you can equip it for your needs by installing packages into it.
+
+## Install packages
+
+Each virtual environment in Python starts with a minimal set of packages that includes:
+
+- The Python Standard Library 
+- pip
+- setuptools
+- wheels
+
+As we saw above, Positron will also install any packages listed in a `requirements.txt`, `pyproject.toml`, or `environment.yml` file that appears in your directory.
+
+But what if you need to install additional packages? You can do this from the Terminal Window with your preferred method. Many Python users will choose to use Python's `pip` module. To use it:
```

Comment:
Maybe mention that Positron requires you to install ipykernel?

Reply:
Done. https://github.com/garrettgman/positron-guides/commit/bbd62a337aed3a758ef790db99b6712c62247015